common/src/main/java/org/opensearch/ml/common/PredictMode.java

```diff
@@ -0,0 +1,18 @@
+package org.opensearch.ml.common;
+
+import java.util.Locale;
+
+public enum PredictMode {
+    PREDICT,
```

Comment:
Seems not a good name. Looks not easy to understand if we tell user "Please use PREDICT PredictMode to do predict"

Reply:
I was thinking to use "realtime", but still end up using this name to ensure bwc with the "actions": [ "action_type": "predict", ...]. We can rename PredictMode to ActionType, but still the values are "predict", "batch", and so on..